drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -57,10 +57,19 @@ class Hmis::Hud::Project < Hmis::Hud::Base
   has_many :households, through: :enrollments
 
   # NOTE: These are not performant, because they go through the WIP view. Don't use them in the application.
-  has_many :custom_assessments, through: :enrollments_including_wip
   has_many :services, through: :enrollments_including_wip
   has_many :custom_services, through: :enrollments_including_wip
 
+  # For now, use a class method instead of an AR association for custom assessments rather than joining thru the
+  # not-very-performant view.
+  # has_many :custom_assessments, through: :enrollments_including_wip
+  def custom_assessments
+    Hmis::Hud::CustomAssessment.joins(:enrollment).left_outer_joins(enrollment: :wip).
+      where(Hmis::Hud::Enrollment.arel_table[:project_id].eq(project_id)).
+      or(Hmis::Hud::CustomAssessment.joins(:enrollment).left_outer_joins(enrollment: :wip).
+        where(Hmis::Wip.arel_table[:project_id].eq(id)))
+  end
+
```

Comment:
For reference, this is output from the test when I tried each approach:
```
  # class method
  SELECT "CustomAssessments".* FROM "CustomAssessments" INNER JOIN "Enrollment" ON "Enrollment"."DateDeleted" IS NULL AND "Enrollment"."EnrollmentID" = "CustomAssessments"."EnrollmentID" AND "Enrollment"."PersonalID" = "CustomAssessments"."PersonalID" AND "Enrollment"."data_source_id" = "CustomAssessments"."data_source_id" LEFT OUTER JOIN "hmis_wips" ON "hmis_wips"."deleted_at" IS NULL AND "hmis_wips"."source_id" = "Enrollment"."id" AND "hmis_wips"."source_type" = $1 LEFT OUTER JOIN "hmis_wips" "wips_Enrollment" ON "wips_Enrollment"."deleted_at" IS NULL AND "wips_Enrollment"."source_id" = "Enrollment"."id" AND "wips_Enrollment"."source_type" = $2 WHERE "CustomAssessments"."DateDeleted" IS NULL AND ("Enrollment"."ProjectID" = '200' OR "hmis_wips"."project_id" = 908936) AND "Enrollment"."DateDeleted" IS NULL AND ("hmis_wips"."project_id" IN (908936) OR "Enrollment"."ProjectID" IN ('200')) LIMIT $3 OFFSET $4

  # relation
  SELECT "CustomAssessments".* FROM "CustomAssessments" INNER JOIN "Enrollment" ON "CustomAssessments"."EnrollmentID" = "Enrollment"."EnrollmentID" AND "CustomAssessments"."PersonalID" = "Enrollment"."PersonalID" AND "CustomAssessments"."data_source_id" = "Enrollment"."data_source_id" INNER JOIN "hmis_client_projects" ON "Enrollment"."id" = "hmis_client_projects"."enrollment_id" INNER JOIN "Enrollment" "enrollments_CustomAssessments" ON "enrollments_CustomAssessments"."DateDeleted" IS NULL AND "enrollments_CustomAssessments"."EnrollmentID" = "CustomAssessments"."EnrollmentID" AND "enrollments_CustomAssessments"."PersonalID" = "CustomAssessments"."PersonalID" AND "enrollments_CustomAssessments"."data_source_id" = "CustomAssessments"."data_source_id" LEFT OUTER JOIN "hmis_wips" ON "hmis_wips"."deleted_at" IS NULL AND "hmis_wips"."source_id" = "Enrollment"."id" AND "hmis_wips"."source_type" = $1 WHERE "CustomAssessments"."DateDeleted" IS NULL AND "hmis_client_projects"."project_id" = $2 AND "Enrollment"."DateDeleted" IS NULL AND ("hmis_wips"."project_id" IN (908939) OR "Enrollment"."ProjectID" IN ('203')) LIMIT $3 OFFSET $4
```

I can't parse this is any meaningful way right now, but, I did a comparison test just for peace of mind to SEE if they really were different:
* Class method: 54ms (1000 records, resolve 50)
* Relation: 60ms (1000 records, resolve 50)

It is a tiny difference in this example, but I'm not sure what the difference would look like at a larger scale.

I would love to have this be done via a relation but I think it will be revisited holistically with [this spike](https://www.pivotaltracker.com/story/show/187373375) . I'll make a note there about this new addition to the list of potentially concerning queries.

So, TLDR, looks good to me 👍